### DISCUSSION TITLE
Is there a way to filter the project menu to have files first?

### DISCUSSION THREAD

**Discussion by LumiFae** [zed-industries/zed#8947] [Feature Requests]
In VS Code, you can set it so your project panel is filtered so files appear first. Is that possible in Zed? Sorry if this question has been asked, couldn't find an answer

**Comment by Moshyfawn** [comment]
The sorting of the project panel is discussed in #4851. At the moment there isn't much you can do to customise the order of the file tree.

**Reply by JosephTLyons** [reply]
Also, there was a proposed PR here, that wasn't merged, but might have some relevant info for anyone wanting to build this feature out:

- https://github.com/zed-industries/zed/pull/9093

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
